### DISCUSSION TITLE
with every change I need to restart the app with "npm start"

### DISCUSSION THREAD

**Discussion by Rushijoshi3798** [remix-run/remix#7274] [Q&A] [answered]
Hey folks! I am new to remix framework.
I faced one issue where every time I change anything It is not rendered on UI but I need to use "npm start" command every time.

Any one faced similar problem & guide me about the same ?

**Comment by MichaelDeBoey** [comment]
CC/ @pcattori

**Comment by kiliman** [comment]
Hmm... which template are you using? Also, which OS?

Do you know if the initial template worked? How much have you changed, especially `remix.config.js`?

Your question could use more details if you want some help.

Thanks!

**Reply by cliffordfajardo** [reply]
@Rushijoshi3798 - what version of Remix are you using?

I created a new remix app using the Remix CLI command below and when I make updates to either the frontend or backend code, things work for me without needing to kill or restart my server manually.

You can use the following command in your terminal below to create a new Remix app:

```sh
npx create-remix@latest my-new-remix-app
```

**Reply by pcattori** [reply]
@Rushijoshi3798 are you using `npm run dev`? If so, what does your `dev` script look like in your `package.json`?

**Comment by Rushijoshi3798** [comment, marked as answer]
Thanks to all, for prompt responses. The issue has been resolved, There was just a small change required in `remix.config.js`

**Reply by braginteractive** [reply]
What was the small change?
